src/Hintman/Suggestion/Core.hs

```diff
@@ -24,7 +22,7 @@ data Suggestion = Suggestion
 data LineChange = LineChange
     { lineChangeNew     :: Text
     , lineChangeComment :: Maybe Text
-    , lineChangeType    :: ChangeType
+    , lineChangeType    :: SuggestionType
```

Comment:
Mm, this is semantically different from what was here before. `ChangeType` is not that important in this case?

Reply:
I guess it was swapped because suggestion is more general then _change_, and can be used for _delete_.